components/spicedb/schema/schema.yaml

```diff
@@ -45,6 +45,9 @@ schema: |-
     relation member: user
     // Some users in an organization may additionally have the `owner` role
     relation owner: user
+    // users who can snapshot workspaces
+    relation snapshoter: user | organization#member
```

Comment:
I'm not sure I understand the intend behind this relation, especially how it is intended to be expanded upon in the future.

The permission `create_snapshot` itself is clear: Owners can take snapshots, if the org allows them to.
The I would expect the relation to either look like:
 - for making it an explicit role on the org which has to be added : `relation snapshoter: organization#member`
   - this would need to be added to every member, though (see comment above)
 - for making it a toggle: `relation snapshoter: user:*`
   - in this case only the `owner &` ensure's that only the owner can do it. I'd rename it to sth like `enable_snapshotting`

Reply:
This allows to enable/disable snapshotting on org level

Reply:
Why keep the `user` there, then?

Reply:
it is not used indeed but since we allow assigning roles to users everywhere I thought I'd do it here as well for consistency reasons. I can remove it if you like that better also because YAGNI

Reply:
> I can remove it if you like that better also because YAGNI
:+1: